### ISSUE TITLE
Odd performance stats

### ISSUE THREAD

**Issue by asmagen** [qubvel/segmentation_models#358]
I'm training a multi-class segmentation model based on adaptation of the tutorial from segmentation_models and the performance stats seem odd in the sense that while the predictions on the test set appear to be pretty good
![Screen Shot 2020-06-14 at 10 35 38 AM](https://user-images.githubusercontent.com/11299568/84596342-1c4f3080-ae2b-11ea-9b31-4aaba5ec4d63.png)
![Screen Shot 2020-06-14 at 10 35 23 AM](https://user-images.githubusercontent.com/11299568/84596332-0d687e00-ae2b-11ea-8235-7ff3a4142e4f.png)
(1) the iou score seems to be better on the validation set relative to the train set, which is unexpected, and
(2) the loss on the validation set seems to not decrease by much, while as expected, the loss on the train set is decreasing.
![Screen Shot 2020-06-14 at 10 35 17 AM](https://user-images.githubusercontent.com/11299568/84596331-0ccfe780-ae2b-11ea-83f7-2e3a48650faa.png)

Any ideas for what might be happening?

**Comment by JordanMakesMaps** [comment]
Do you think it could be how you're splitting your data up? (1) implies underfitting, so are you applying a lot of augmentation?
